thirdparty/flac/CMakeLists.txt

```diff
@@ -71,6 +71,8 @@ set(MODULE_USE_PCH_NONE ON)
 set(MODULE_USE_UNITY_NONE ON)
 include(${PROJECT_SOURCE_DIR}/build/module.cmake)
 
+target_no_warning(${MODULE} -Wno-deprecated)
+target_no_warning(${MODULE} -Wno-conversion)
```

Comment:
Strangely this doesnt help, the 2 warnings still show:
Warning	C4244	'=': conversion from 'const uint32_t' to 'FLAC__byte', possible loss of data	...\thirdparty\flac\flac-1.4.3\src\libFLAC\metadata_iterators.c	2725
Warning	C4244	'=': conversion from 'uint32_t' to 'FLAC__byte', possible loss of data	...\thirdparty\flac\flac-1.4.3\src\libFLAC\metadata_iterators.c	2877